src/DataProtection/DataProtection/src/AuthenticatedEncryption/ISpanAuthenticatedEncryptor.cs

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+#if NETCOREAPP
+using System;
+
+namespace Microsoft.AspNetCore.DataProtection.AuthenticatedEncryption
+{
+    internal interface ISpanAuthenticatedEncryptor : IAuthenticatedEncryptor
```

Comment:
Note: this is internal, not sure if we want to expose this publically, we have another IOptomizedAuthenticatedEncryptor which also is internal so I started with making this internal as well.